src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -35,6 +35,10 @@
     width: calc(100% - #{2 * $gui-gutter}px);
     height: calc(100% - #{$bottom-bar-bottom-gutter}px);
     margin: 0 #{$gui-gutter}px #{$bottom-bar-bottom-gutter}px;
+    pointer-events: none;
+    * {
```

Comment:
as discussed need to make it for direct children only

Reply:
Solved in FEC-11359 - closing this PR